instructions/index.md

```diff
@@ -0,0 +1,14 @@
+# Instructions
+
+- Please, navigate to [`src/template`](../src/template) folder and change its name to your first and lastname
```

Comment:
They should probably copy and paste it or we'll lose the template when the PR is merged

Reply:
This could also be the README